bootloader/bootloadertest/bootloadertest.go

```diff
@@ -45,10 +45,19 @@ type MockBootloader struct {
 	InstallBootConfigResult bool
 	InstallBootConfigErr    error
 
+	enabledKernel    snap.PlaceInfo
+	enabledTryKernel snap.PlaceInfo
+
 	panicMethods map[string]bool
 }
 
-// ensure MockBootloader(s) implement the Bootloader interfaceces
+type Mock20BootloaderSetup interface {
```

Comment:
no, it can't be boottest.MockUC20RunBootenv because that already exists and is a function. What's wrong with Mock20BootloaderSetup ? the interface is not specific to run mode as far as I can tell?

Reply:
I will move the interface definition to boottest as you prefer however

Reply:
actually I see is just used in one place atm, it might be easier to define a local interface there for now

Reply:
sure I can do that too